docs/source/contributing/development.rst

```diff
@@ -235,7 +235,7 @@ Further useful guidelines
    frequently.
 
 #. When opening a new issue, if there are old issues that are related, link
-   them in your new issue (even if the old ones are closed).
+   them to your new issue (even if the old ones are closed).
```

Comment:
"link them in" is correct.

Reply:
But grammatically "link them to" is correct

Reply:
in.

You make a link to the old issue in your new issue. So what you really want is "Add a link **to** them **in** your new issue"

Reply:
Changed to the comment below